tests/test_code_repr.py

```diff
@@ -378,3 +378,42 @@ def __eq__(self,other):
 """
         )
     )
+
+
+def test_function_type():
+
+    Example(
+        """\
+from inline_snapshot import snapshot
+from types import FunctionType
+def func():...
+def test():
+    assert func == snapshot()
+"""
+    ).run_inline(  # run without flags
+        reported_categories=snapshot(["create"]),
+    ).run_pytest(
+        ["--inline-snapshot=short-report"],  # check the pytest report
+        changed_files=snapshot({}),
+        report=snapshot(
+            """\
+Error: one snapshot is missing a value (--inline-snapshot=create)
+You can also use --inline-snapshot=review to approve the changes interactively\
+"""
+        ),
+        returncode=snapshot(1),
+    ).run_pytest(  # run with create flag and check the changed files
+        ["--inline-snapshot=create"],
```

Comment:
I think you can remove the first two tests (category and short-report). This is basic behavior which is already tested. Keep the last test where you test `--inline-snapshot=create`.
```suggestion
"""
    ).run_pytest(  # run with create flag and check the changed files
        ["--inline-snapshot=create"],
```